src/main/java/com/zeepy/server/review/service/ReviewService.java

```diff
@@ -1,35 +1,38 @@
 package com.zeepy.server.review.service;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
 import com.zeepy.server.common.CustomExceptionHandler.CustomException.NoContentException;
 import com.zeepy.server.review.domain.Review;
+import com.zeepy.server.review.dto.ReviewDto;
 import com.zeepy.server.review.dto.ReviewResponseDto;
 import com.zeepy.server.review.dto.ReviewResponseDtos;
-import com.zeepy.server.review.dto.ReviewDto;
 import com.zeepy.server.review.repository.ReviewInterface;
-import lombok.RequiredArgsConstructor;
-import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
-import java.util.List;
-import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
 
 @Service
 @RequiredArgsConstructor
 public class ReviewService {
-    private final ReviewInterface reviewInterface;
+	private final ReviewInterface reviewInterface;
 
-    @Transactional(readOnly = true)
-    public ReviewResponseDtos getReviewList(String address) {
-        List<Review> reviewList = reviewInterface.findAllByAddress(address);
-        if (reviewList.isEmpty()) throw new NoContentException();
-        return new ReviewResponseDtos(reviewList.stream()
-                .map(ReviewResponseDto::new)
-                .collect(Collectors.toList()));
-    }
+	@Transactional(readOnly = true)
+	public ReviewResponseDtos getReviewList(String address) {
+		List<Review> reviewList = reviewInterface.findAllByAddress(address);
+		if (reviewList.isEmpty())
+			throw new NoContentException();
```

Comment:
```suggestion
		if (reviewList.isEmpty()) {
			throw new NoContentException();
		}
```
왠만하면 중괄호를 생략하지 않는 것을 추천해요. 지금은 이 조건문에 추가될 코드가 없을 가능성이 높지만, 만약 코드가 추가된다면 중괄호를 작성해줘야 합니다. 그 중괄호를 작성 해주지 않는다면 오류가 발생하겠죠. 오류에 대한 발생 가능성을 최대한 줄여주는 것이 좋은 코드라고 생각하는데, 어떻게 생각하시나요?